pennylane/labs/tests/trotter_error/product_formulas/test_bch.py

```diff
@@ -319,3 +320,29 @@ def test_effective_hamiltonian_backend(backend, num_workers, mpi4py_support):
     expected *= 1j / r * delta
 
     assert np.allclose(1j * delta * (expected + ham), actual)
+
+
+def test_importance_scores():
```

Comment:
Maybe I missed, but where in the docs we explain how we determine that a commutator is not important?